core/templates/pages/admin-page/misc-tab/admin-misc-tab.directive.ts

```diff
@@ -19,16 +19,17 @@
 require('domain/utilities/url-interpolation.service.ts');
 require('pages/admin-page/services/admin-data.service.ts');
 require('pages/admin-page/services/admin-task-manager.service.ts');
+require('pages/admin-page/services/admin-misc-tab-backend-api.service');
```

Comment:
Add this in alphabetically sorted order.

Reply:
done!

Reply:
Removed separate misc-tab-backend-api-service as you have mentioned